docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst

```diff
@@ -310,6 +304,131 @@ partition from *Talker 2* and so it does not listen to it.
     listen to each other messages.
 
 
+ROS 2 Introspection
+-------------------
+
+ROS 2 Command Line Interface (CLI) implements several introspection features to analyze the behaviour of a ROS2
+execution.
+These features (i.e. `rosbag`, `topic list`, etc.) are very helpful to understand a ROS 2 working network.
+
+Most of these features use the DDS capability to share any topic info with every exiting participant.
+However, the new :ref:`ros2_discovery_server_v2` implements a traffic network reduction
+that limits the discovery data between nodes that do not share a topic.
+This means that not every node will receive every topic data unless it has a reader in that topic.
+
+The Discovery Server v2 functionality allows every server to know and share all the participants and topic information
+with every other server matched.
+So, this functionality can be used in order to use ROS 2 introspection successfully when running with Discovery Server
+discovery protocol.
```

Comment:
```suggestion
In this sense, a server can be configured alongside ROS 2 introspection, since then the introspection tool will be able to discover every entity in the network that is using the Discovery Server protocol.
```

Reply:
:+1: